src/client.cpp

```diff
@@ -758,8 +734,18 @@ void CClient::OnSndCrdReinitRequest ( int iSndCrdResetType )
     }
     MutexDriverReinit.unlock();
 
+    if ( !strError.isEmpty() )
+    {
+#ifndef HEADLESS
+        QMessageBox::critical ( 0, APP_NAME, strError, tr ( "Ok" ) );
+#else
+        qCritical() << qUtf8Printable ( strError );
```

Comment:
Not great. We may want to use the old error message handling for now and than change this in future.

Reply:
I'd like all this removed from the PR.  I can't see how it relates.